tools/shared.py

```diff
@@ -903,14 +905,18 @@ def set_logging():
 
 # Target choice.
 ASM_JS_TARGET = 'asmjs-unknown-emscripten'
-WASM_TARGET = 'wasm32-unknown-unknown'
+WASM_TARGET = 'wasm32-unknown-unknown-elf'
+WASM_OBJ_TARGET = 'wasm32-unknown-unknown-wasm'
 
```

Comment:
this surprises me - a different triple just for the feature of using wasm object files?